deployment/stacks/api.ts

```diff
@@ -116,9 +107,6 @@ export function createApiComponents(props: CreateApiComponentsProps) {
       tables.map((table) => [`${table.id}Table`, table.name])
     ),
     BOOTSTRAP_BROKER_STRING_TLS: brokerString,
-    COGNITO_USER_POOL_ID: userPoolId || process.env.COGNITO_USER_POOL_ID,
```

Comment:
I thought we added this to get the parameters out of SSM, we don't need this anymore?

Reply:
Because the api already does the authentication so checking that the api token matches the cognito params is redoing on every single request what cognito just did before it let the request into the lambda. QMR was already set up like this.